proof-producer/libs/assigner/src/trace_parser.cpp

```diff
@@ -149,5 +156,94 @@ namespace nil {
 
             return zkevm_states;
         }
+
+
+        namespace {
```

Comment:
namespace required here?

Reply:
For our libraries with header-only files it is not needed, but for rare cases of non-testing .cpp files it is better to declare unexported functions as static or put them into an anonymous ns (which is the same) to prevent potential symbol resolution issues during linking

Reply:
Yes, done